### PR TITLE
log initialization to a file

### PR DESCRIPTION
**Pull request by gares**

### CODE REVIEW

File: language-server/vscoqtop/vscoqtop.ml [hunk 1]
```diff
@@ -46,20 +46,34 @@ let vscoqtop_specific_usage = {
   extra_options = "";
 }
 
+let init_log =
+  (* we put the time to make the file names easy to sort *)
+  try Some (open_out @@ Filename.temp_file (Printf.sprintf "vscoq_init_log.%f." (Unix.gettimeofday())) ".txt")
```
Comment (maximedenes) [inline language-server/vscoqtop/vscoqtop.ml:51]: I don't think it is useful to put the time in the name, filesystems already expose it, don't they?

Comment (gares) [reply language-server/vscoqtop/vscoqtop.ml:51]: I never recall how to sort by creation time with ls, while I recall I can do ls|sort

Comment (maximedenes) [reply language-server/vscoqtop/vscoqtop.ml:51]: `ls -ltr` ;)

Comment (gares) [inline language-server/vscoqtop/vscoqtop.ml:51]:
```suggestion
  try Some (open_out @@ Filename.temp_file "vscoq_init_log." ".txt")
```

**Comment by gares** [comment]
@maximedenes ping

**Pull request closed without merging**